src/hawkbit-client.c

```diff
@@ -598,6 +598,7 @@ static gboolean process_deployment(JsonNode *req_root, GError **error)
 {
         GError *ierror = NULL;
         struct artifact *artifact = NULL;
+        g_autofree gchar *str;
```

Comment:
Note that in order to have autocleanup working correctly, this needs to be initialized to NULL.
```suggestion
        g_autofree gchar *str = NULL;
```

Reply:
:man_facepalming:   Sorry.  I saw  the implementation in other points and then... I forget to put the NULL :man_facepalming: :man_facepalming: .  Do I push a fixup with it or I directly do a forced push ?

Reply:
just do a force-push